index.ios.js

```diff
@@ -115,7 +118,10 @@ var MapView = React.createClass({
       showsUserLocation: false,
       styleUrl: this.Mixin.mapStyles.streets,
       zoomEnabled: true,
-      zoomLevel: 0
+      zoomLevel: 0,
+      attributionButtonIsHidden: false,
+      logoIsHidden: false,
+      compassIsHidden: false
```

Comment:
It’d be less confusing to spin these properties in a positive light, e.g. `showsLogo`. (See `showsUserLocation` above.)